README.md

```diff
@@ -10,4 +10,4 @@ a folder within the application directory.
 * Go into the SCATT directory with the command `cd cs-5666-scatt-gp`.
 * Enter `make scatt` and hit return.
 * Enter the name of your SCATT submissions folder when prompted.
-* View the report output.
+* To view the report, open the file named `Report.txt`.
```

Comment:
@KaraBeason @caldwellc1 Although they will already be in the directory, this is probably a good idea to let them know they can find/view Report.txt from their current location. Also, I like the idea of adding the confirmation message to stdout, so I added an issue for our Iteration 3 project for this.